From 689100b259e96ccb9540e5b0b3c283c9dee342a8 Mon Sep 17 00:00:00 2001
From: Ting Liu <b28495@freescale.com>
Date: Tue, 21 May 2013 11:15:26 +0800
Subject: [PATCH] install perl lib to directory perl instead of perl5

libhugetlbfs depends on perl, and perl installs a directory 'perl'
and make a symlink 'perl5' to it. So just install perl libs in
libhugetlbfs to directory 'perl' instead of 'perl5'. This can avoid
the following error:
| CalledProcessError: Command 'tar -cf - -C .../perl/5.14.3-r1/sysroot-destdir
| -ps . | tar -xf - -C .../tmp/sysroots/t4240qds'
| returned non-zero exit status 2 with output
| tar: ./usr/lib/perl5: Cannot create symlink to `perl': File exists
| tar: Exiting with failure status due to previous errors

Upstream-Status: Inappropriate [oe-core specific]

Signed-off-by: Ting Liu <b28495@freescale.com>
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index a7c7551..8de2a8f 100644
--- a/Makefile
+++ b/Makefile
@@ -151,9 +151,9 @@ BINDIR = $(PREFIX)/share/libhugetlbfs
 EXEDIR = $(PREFIX)/bin
 DOCDIR = $(PREFIX)/share/doc/libhugetlbfs
 ifdef CC32
-PMDIR = $(PREFIX)/lib/perl5/TLBC
+PMDIR = $(PREFIX)/lib/perl/TLBC
 else
-PMDIR = $(PREFIX)/lib64/perl5/TLBC
+PMDIR = $(PREFIX)/lib64/perl/TLBC
 endif
 MANDIR1 = $(PREFIX)/share/man/man1
 MANDIR3 = $(PREFIX)/share/man/man3
-- 
1.7.5.4

